monitoring/views/database_sizes_mb.view.lkml

```diff
@@ -0,0 +1,58 @@
+# If necessary, uncomment the line below to include explore_source.
+include: "//looker-hub/fenix/explores/metrics.explore.lkml"
```

Comment:
instead of extending the metrics explore, it would have been a little more straightforward to extend the metrics view and define `database_size_sum_mb` and `database_size_tiers` as dimensions.

Reply:
good to know. will keep that in mind